README.md

```diff
@@ -15,7 +15,7 @@ npm install -g @elastic/synthetics
 Run the suites via CLI
 
 ```
-npx @elastic/synthetics [options] [dir] [files] file
+cd my-test-dir; npx @elastic/synthetics [options] [dir] [files] file
```

Comment:
We should not have this on the main readme page. It is handled via directory option.